### DISCUSSION TITLE
Unable to install now

### DISCUSSION THREAD

**Discussion by DanielMichaelEdwards** [vercel/vercel#4025] [General] [answered]
I tried to update my now version and I got this error:

![image](https://user-images.githubusercontent.com/18138043/78796153-3d704c80-79b6-11ea-954e-ca23abaf590c.png)

I have tried to uninstall it and reinstall it and I get the same error. Is there a problem on the Now deployment?

**Comment by styfle** [comment, marked as answer]
The `now` package doesn't depend on `eslint-plugin-flowtyp` or any other dependencies.

Maybe you have a typo in your package.json? Should `eslint-plugin-flowtyp` be `eslint-plugin-flowtype`?

Also you could look in the debug log file that was generated in your screenshot.
